### PR TITLE
support for venv system packages

### PR DESCRIPTION
**Pull request by h0tw1r3**
Adds parameter to support system packages in venv.

### CODE REVIEW

**Review by ghoneycutt — APPROVED** [review]
Given the trivial nature of the commit, that it does not introduce new conditionals and that the current spec tests are so minimal, I think this is fine to merge as is once it is rebased.

If there were already some spec tests around an existing param, I would hold up merging and say copy/paste the existing one and change it for the new param, but that's not the case here.

**Comment by ghoneycutt** [comment]
@h0tw1r3 Could you please rebase? That will allow this to be merged.

**Comment by h0tw1r3** [comment]
rebased

**Comment by h0tw1r3** [comment]
rebased again

File: manifests/init.pp
Comment (smortex) [inline manifests/init.pp:85]: Maybe we should not change the default value a user might have set globally with:

```puppet
Python::Pyvenv {
  systempkgs => true,
}
```

…and do not set an explicit value unless one is passed:

```suggestion
  Optional[Boolean] $python_systempkgs                        = undef,
```

**Comment by smortex** [comment]
> Given the trivial nature of the commit, that it does not introduce new conditionals and that the current spec tests are so minimal, I think this is fine to merge as is once it is rebased.

Same feeling here.  I am removing the needs-test label.  If people are not happy with this, feel free to re-add the label ;-)

File: manifests/init.pp
Comment (smortex) [reply manifests/init.pp:85]: Also wondering if the name should not rather contain `pyvenv` since it applies only to `python::venv` and we have other `python` things… What do you think @h0tw1r3 ?

Comment (h0tw1r3) [reply manifests/init.pp:85]: I don't actually use puppetboard anymore, just rebasing this to get it off my list :)

A default of `false` makes the most sense to me because it's the current behavior.
The variable naming follows the current convention.

I could make sense to review and change both of these, but that would be a breaking change.
This pull request has a very limited scope.

Comment (smortex) [reply manifests/init.pp:85]: w00t, my bad, I missed the previous value in the diff! You are right, this would change the default!

**Review by smortex — APPROVED** [review]
LGTM!

**Pull request merged**
